tools/vms-generator/utils/utils.go

```diff
@@ -99,6 +100,24 @@ const defaultInterfaceName = "default"
 const enableNetworkInterfaceMultiqueueForTemplate = true
 const EthernetAdaptorModelToEnableMultiqueue = "virtio"
 
+const (
+	cloudConfigHeader = "#cloud-config"
+
+	cloudConfigInstallAndStartService = `packages:
+  - nginx
+runcmd:
+  - [ "systemctl", "enable", "--now", "nginx" ]`
+
+	cloudConfigUserPassword = `password: fedora
+chpasswd: { expire: False }`
+
+	secondaryIfaceDhcpNetworkData = `version: 2
+ethernets:
+  eth1:
+    dhcp4: true
+`
```

Comment:
Had to make it like this (yes,  **exactly** like this) to preserve the `|-` on the examples. 

Not sure if it looks that better than without the `\n` s. Your thought @ormergi ?

Reply:
I think its better, its easier to understand comparing to one line with `\n` :slightly_smiling_face: 
nit: I think better naming would be`cloudConfigInstallAndStartService` -> `cloudConfigInstallAndStartNginx`